### DISCUSSION TITLE
SecuritySystem Apple  Home Notification Doesn't show up

### DISCUSSION THREAD

**Discussion by Stequila93** [HomeSpan/HomeSpan#838] [Q&A]
Hello,
I have an esp32 with a SecuritySystem accessory, I was getting notification of change status from Apple Home app untill a month ago that maybe i updated to iOs 17 my ipad and lost the capability to act like a home hub.
So i bought a homepod, but i'm not getting any kind of notification when status switches.
Does iOs 17 support notification from not original acessory like homespan? Should i do something different in code?

Thanks have a nice day.

**Comment by HomeSpan** [comment]
Apple discontinued the use of an iPad as a Home Hub, so switching to a HomePod should work.  There is nothing in the HomeSpan code that needs to change.  Have you tried unpairing and re-pairing the HomeSpan device so that the HomePod can make a fresh connection?   Also, check to make sure your HomeSpan device and the HomePod are on the same WiFi network --- if your router uses both 5GHz and 2.4 GHz, it needs to be configured to allow cross-traffic between the two.

**Reply by Stequila93** [reply]
Thanks,
I just checked that both are on the same wifi, yes i have both 2.4 an 5 GHz but i didnt find out any "cross-traffic" by the way i took a new esp32 with the same sw and added as a new accessories on the same network  of homepod  and still doesnt show up any notification but it's working in the home app.
I'm on a 17.5 iOs version does it matter?, last one on homepod but not any beta version, I also have an other iphone that should not be on beta version and doesnt work on that also.
Also notification status in the setting of the device are on always and nobody.

**Comment by HomeSpan** [comment]
The issue could be related to 17.5.   I have found that the HomePod and iPhone used for pairing must be 100% in sync in terms of iOS version, else notifications or other problems arise.   Also, the beta version of iOS almost always breaks something in HomeKit, which usually (though not for certain) gets resolved in the production version.

Re adding a second ESP32 with the same software, did you make sure the device display names are different (see this [FAQ](https://github.com/HomeSpan/HomeSpan/blob/master/docs/FAQ.md#can-you-use-more-than-one-homespan-device-on-the-same-homekit-network)).

**Reply by Stequila93** [reply]
Something strange is happening.
I updated an iphone to 17.4.1 because it was on 16. 
Then i tried the new esp added to home with that iphone at 17.4.1 and i got a notification on my iphone 17.5 . 
So i checked the other one installed and it's working just on my iphone now.
Now i'm getting notifications on 17.5 but not on 17.4.1 so maybe i need to wait the release of 17.5 for all devices and maybe it will work?
This means that if one device keep update over others i will lose comatibility for those "old" device?

**Reply by HomeSpan** [reply]
In general you can't mix and match different versions of iOS with homekit. My guess is this will resolve once 17.5 is out of beta and you can deploy across all devices.
